Package.swift

```diff
@@ -44,6 +44,10 @@ let package = Package(
       name: "FirebaseDatabase",
       targets: ["FirebaseDatabase"]
     ),
+    .library(
+      name: "FirebaseDatabaseSwift",
```

Comment:
Ah, one last thing! I forgot that we moved to append a `-Beta` to the Swift libs in SwiftPM:

https://github.com/firebase/firebase-ios-sdk/blob/f5fbdac038ac6fa284074ea2a101f22a573bbb87/Package.swift#L86-L89

After that I'm good to merge this and make any API changes as necessary from the API review in a follow up PR. Thank you!

Reply:
Done! Looking forward to the merge. Thank you so much for all your help! :-)